Example/ViewController.swift

```diff
@@ -186,6 +187,14 @@ class ViewController: UIViewController {
         
         setupGestureRecognizers()
         setupPerformActionBarButtonItem()
+        if let lastKnownActiveNavigationLocation = lastKnownActiveNavigationLocation {
+            /// Empirical zoom level that produce no zoom animation after active navigation dismiss
+            let zoom = 14.375349173038527
```

Comment:
Why not remember last known `CameraOptions`? (it'll include both last known coordinate and zoom level)

Reply:
Thanks for suggestion, I'll try

Reply:
Done